service/src/alive_service.rs

```diff
@@ -0,0 +1,55 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+//! Service to determine if the integritee services is alive and registered to the node,
```

Comment:
```suggestion
//! Service to determine if the integritee services is alive and registered on the node,
```

Reply:
Maybe it's more accurate to name it `is_initialized` (and shorter than `has been successfully started`) ?